deploy/docs/Non_Helm_Installation.md

```diff
@@ -60,273 +39,153 @@ First, you'll need to set up the relevant [fields](https://help.sumologic.com/Ma
-* **HTTP Source URLs.** When you configure each HTTP Source, Sumo will display the URL of the HTTP endpoint. Make a note of the URL. You will use it when you configure the Kubernetes service secrets to send data to Sumo.
-
-#### Create the namespace and secret
-
-Run the following command to create namespace `sumologic`
-
-```sh
-kubectl create namespace sumologic
-```
-
-Run the following command to create a Kubernetes secret containing the 9 HTTP source URLs previously created.
-
-```sh
-kubectl -n sumologic create secret generic sumologic \
-  --from-literal=endpoint-metrics=$ENDPOINT_METRICS \
-  --from-literal=endpoint-metrics-apiserver=$ENDPOINT_METRICS_APISERVER \
-  --from-literal=endpoint-metrics-kube-controller-manager=$ENDPOINT_METRICS_KUBE_CONTROLLER_MANAGER \
-  --from-literal=endpoint-metrics-kube-scheduler=$ENDPOINT_METRICS_KUBE_SCHEDULER \
-  --from-literal=endpoint-metrics-kube-state=$ENDPOINT_METRICS_KUBE_STATE \
-  --from-literal=endpoint-metrics-kubelet=$ENDPOINT_METRICS_KUBELET \
-  --from-literal=endpoint-metrics-node-exporter=$ENDPOINT_METRICS_NODE_EXPORTER \
-  --from-literal=endpoint-logs=$ENDPOINT_LOGS \
-  --from-literal=endpoint-events=$ENDPOINT_EVENTS
-```
-
-## Deploy Fluentd
-
-In this step you will deploy Fluentd using a Sumo-provided .yaml manifest. 
-
-### Use default configuration
-
-If you don't need to customize the configuration apply the `fluentd-sumologic.yaml` manifest with the following command. Replace the `<NAMESPACE>` and `<CLUSTER_NAME>` variables with your values.
-
-```sh
-curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/release-v1.0/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
-sed 's/\$NAMESPACE'"/<NAMESPACE>/g" | \
-sed 's/cluster kubernetes/cluster <CLUSTER_NAME>/g' | \
-kubectl -n sumologic apply -f -
-```
-
-### Customize configuration
-
-If you need to customize the configuration there are two commands to run. First, get the `fluentd-sumologic.yaml` manifest with following command:
-
-```sh
-curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/release-v1.0/deploy/kubernetes/fluentd-sumologic.yaml.tmpl | \
-sed 's/\$NAMESPACE'"/sumologic/g" >> fluentd-sumologic.yaml
-```
+The following parameter is optional, but we recommend setting it.
+* __sumologic.clusterName__ - An identifier for your Kubernetes cluster.  This is the name you will see for the cluster in Sumo Logic. Default is `kubernetes`.
 
-Next, customize the provided YAML file. Our [plugin](../../fluent-plugin-events/README.md#fluent-plugin-events) allows you to configure fields for events. Once done run the following command to apply the `fluentd-sumologic.yaml` manifest.
+First, you will need to apply the required CRD's for the Prometheus Operator. This is required before generating the YAML.
```

Comment:
actually this is required before the applying the generated template

Reply:
Good catch. Will change order